test/function/samples/tree-shaking-within-if-statement/_config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = defineTest({
```

Comment:
This should probably be a form test so that we can actually see what is removed?

Reply:
That makes sense!